### PR TITLE
Feature/profile part 1

### PR DESCRIPTION
**Pull request by Kacym**

### CODE REVIEW

File: src/assets/index.js [hunk 1]
```diff
@@ -16,4 +16,16 @@ export { ReactComponent as SuccessIcon } from './icons/notify-success-icon.svg'
 export { default as FriendsMakeASelphyImage } from './images/friends-make-a-selphy.png'
 export { default as GirlsImage } from './images/one-girl-with-glasses-and-two-girl-without-glasses.png'
 export { default as ArrowDown } from './icons/arrow_down.svg'
+<<<<<<< HEAD
```
Comment (Satybaldyeva) [inline src/assets/index.js]: bul jerde conflict kalyp kalyptyr

File: src/assets/index.js [hunk 2]
```diff
@@ -16,4 +16,16 @@ export { ReactComponent as SuccessIcon } from './icons/notify-success-icon.svg'
...
+
+// user profile social media links
+
+export { default as ProfileFacebook } from './icons/profile_facebook.svg'
```
Comment (Satybaldyeva) [inline src/assets/index.js]: svg formatty ReactComponent dep export kylabyz

File: src/assets/index.js [hunk 3]
```diff
@@ -16,4 +16,16 @@ export { ReactComponent as SuccessIcon } from './icons/notify-success-icon.svg'
...
+export { default as ProfileInstagram } from './icons/profile_instagram.svg'
+export { default as ProfileTelegram } from './icons/profile_telegram.svg'
+export { default as ProfileVk } from './icons/profile_vk.svg'
+
+export { default as ProfilePicture } from './images/profile-picture.png'
+=======
 export { ReactComponent as UploadImageIcon } from './icons/upload-image-icon.svg'
+>>>>>>> fe7752e1f4e30bccf59c6ee335c8def2922a2699
```
Comment (NuraiymMamatova) [inline src/assets/index.js]: Clean this file

**Review by Satybaldyeva — APPROVED** [review]

**Review by NuraiymMamatova — APPROVED** [review]

File: src/pages/LandingPage/Profile.jsx [hunk 4]
```diff
@@ -1,5 +1,345 @@
...
+      clothSize: 'S',
+      shoesSize: '36',
+   },
+]
+
+export const Profile = ({ variant }) => {
+   return (
+      <ProfileContainer component="div">
+         {userInfo.map((user) => (
+            // eslint-disable-next-line react/jsx-fragments
```
Comment (AvazMAX) [inline src/pages/LandingPage/Profile.jsx]: eslint settings cannot be ignored

**Comment by AvazMAX** [comment]
image

something is wrong with him

**Pull request merged**